app/views/api/index.html.erb

```diff
@@ -210,3 +262,117 @@ curl -H 'Littlesis-Api-Token: RhS3mQneriAaym4pNjdjw' https://littlesis.org/api/e
     </div>
 </div>
 
+
+<div class="row">
+  <%= api_column do %>
```

Comment:
this is cool!

Reply:
thanks! i found it really helpful to write the docs first!